client/memory.c

```diff
@@ -44,14 +44,15 @@ int xdag_free_all(void)
 #include <sys/mman.h>
 #include <errno.h>
 
-#define MEM_PORTION     ((size_t)1 << 25)
+#define MEM_PORTION     	((size_t)1 << 25)
+#define TMPFILE_TEMPLATE 	"xdag-tmp-XXXXXX"
 
 static int g_fd = -1;
 static size_t g_pos = 0, g_fsize = 0, g_size = 0;
 static void *g_mem;
 static pthread_mutex_t g_mem_mutex = PTHREAD_MUTEX_INITIALIZER;
 static char g_tmpfile_path[1024] = "";
-static char g_tmpname[64] = "xdag-tmp-XXXXXX";
+static char g_tmpfile_to_remove[1088];
```

Comment:
Why 1088? :)

Reply:
it is 1024+64 :D

Reply:
Is the g_tmpfile_path user now?

Reply:
Sorry, typo. "used"

Reply:
no

Reply:
Ah, yes it is used when you launch pool with -z path